contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
I would strongly prefer using official packages whenever possible but I'm realistic, `calng` case shows that this can be problematic so if there will be need I'll maintain two sets of spec files - one for upstream and one for Fedora.